app/react/kubernetes/namespaces/components/ResourceQuotaFormSection/ResourceQuotaFormSection.tsx

```diff
@@ -57,13 +57,24 @@ export function ResourceQuotaFormSection({
           <div className="flex flex-row">
             <FormSectionTitle>Resource Limits</FormSectionTitle>
           </div>
+
+          {(!cpuLimit || !memoryLimit) && (
```

Comment:
Is this linked with disabling the create / edit button?

Reply:
I think so. If there is no resources available we disable the buttons already